scalableminds/webknossos
# Markdown annotation description

This PR adds markdown rendering for the annotation description. I reused the "Edit markdown" modal from the comment tab to get a live preview when editing the annotation description. This PR also adds appropriate tooltips to the Edit and Save icons of the EditableTextLabel, and only shows the description icon in the explorative annotations list iff there is a description.

### URL of deployed dev instance (used for testing):
- https://markdowndescription.webknossos.xyz

### Steps to test:
- Edit the description of a tracing, it should render markdown correctly
- Download tracing and upload (using upload annotation on dashboard), the annotation description should still contain markdown and line breaks
- Open the explorative annotations list. The annotations with a description should show a clickable icon next to the "rename" icon. Clicking on the description icon should show the description rendered as markdown.

### Issues:
- fixes #3006 

------
